Validation/Geometry/test/MaterialBudget.py

```diff
@@ -836,38 +837,41 @@ def setRanges(h):
         hist_X0_elements[label].SetFillColor(color)
         hist_X0_elements[label].SetLineColor(kBlack)
         stack_X0.Add(hist_X0_elements[label])
-        theLegend.AddEntry(hist_X0_elements[label], leg, "f")
+        if hist_X0_elements[label].Integral() > 0.: theLegend.AddEntry(hist_X0_elements[label], leg, "f")
 
     # canvas
     canname = "MBCan_1D_%s_%s"  % (detector, plot)
     can = TCanvas(canname, canname, 800, 800)
     can.Range(0,0,25,25)
+    can.SetTopMargin(0.05)
```

Comment:
Please consider moving this canvas settings to `plot_utils` [here](https://github.com/cms-sw/cmssw/blob/1d77d131816b1907dbf11e99949558729e4819d4/Validation/Geometry/python/plot_utils.py#L129).

Reply:
Hi @vargasa , i moved the canvas margin settings to plot_utils. The SetOptTitle is still there because some other plots in the macro need it set to 1 (instead of 0).